src/storage.rs

```diff
@@ -590,11 +700,11 @@ impl Session {
             return Ok(IncomingEntry { 
                 id: signature_with_key.id.clone(), 
                 operation: message_obj.operation, 
-                public_key: message_obj.public_key 
+                value: message_obj.value 
```

Comment:
If we are using serde_json for value, meaning value must be valid JSON, we should not have the fields in our structs  be Strings but the respective JSON type. This can be done in a followup PR but open an issue for it.

Alternatively we could find a way to not enforce it to be JSON. This allows applications to use less bytes on the DA layer, which is important because DA is expensive and we pay for every byte. Maybe we can just take some type that implements Serialize/Deserialize and then call the respective methods here.